api/package.json

```diff
@@ -87,6 +87,81 @@
 		"@directus/schema": "workspace:*",
 		"@directus/shared": "workspace:*",
 		"@directus/specs": "workspace:*",
+		"@godaddy/terminus": "^4.10.2",
+		"@rollup/plugin-alias": "^3.1.9",
+		"@rollup/plugin-virtual": "^2.1.0",
+		"argon2": "^0.29.1",
+		"async": "^3.2.4",
+		"async-mutex": "^0.3.2",
+		"axios": "^0.27.2",
+		"busboy": "^1.6.0",
+		"bytes": "^3.1.2",
+		"camelcase": "^6.2.0",
+		"chalk": "^4.1.1",
+		"chokidar": "^3.5.3",
+		"commander": "^8.0.0",
+		"cookie-parser": "^1.4.6",
+		"cors": "^2.8.5",
+		"csv-parser": "^3.0.0",
+		"date-fns": "^2.28.0",
+		"deep-diff": "^1.0.2",
+		"deep-map": "^2.0.0",
+		"destroy": "^1.2.0",
+		"dotenv": "^10.0.0",
+		"encodeurl": "^1.0.2",
+		"eventemitter2": "^6.4.5",
+		"execa": "^5.1.1",
+		"exifr": "^7.1.3",
+		"express": "^4.18.1",
+		"fast-redact": "^3.1.1",
+		"flat": "^5.0.2",
+		"fs-extra": "^10.1.0",
+		"globby": "^11.0.4",
+		"graphql": "^15.5.0",
+		"graphql-compose": "^9.0.8",
+		"helmet": "^4.6.0",
+		"inquirer": "^8.2.4",
+		"ioredis": "^5.2.3",
+		"joi": "^17.6.0",
+		"js-yaml": "^4.1.0",
+		"js2xmlparser": "^4.0.2",
```

Comment:
What happened here? For some reason all these dependencies show up as new 🤔 

Also lets make sure we drop the ranges from all these 👍🏻

Reply:
Moving to a new branch so this is taken care of as seems odd it would do this on merge so easier to start over.

Reply:
It looks be resulting from this manual merge [26b51f8](https://github.com/directus/directus/pull/15367/commits/26b51f8ab02078811f328cc552a8592b2fda6e4b).
An alternative to a new branch could also be to force push back to [c8918a1](https://github.com/directus/directus/pull/15367/commits/c8918a1080089a7d1529f21ac204f0edb17dd68a).
Either way works. 🙂